python/cm/bundle.py

```diff
@@ -299,32 +306,52 @@ def cook_roles(bundle):
                     'prototype__bundle_id': bundle.id,
                 },
             },
+            parametrized_by_type=act.prototype.type,
```

Comment:
`parametrized_by_type` is a `list`

Reply:
fixed

Reply:
and yes, hanging comma is bad replacement for explicit list literal